core-services/image-mirroring/openshift/mapping_origin_4_3

```diff
@@ -185,6 +185,7 @@ registry.svc.ci.openshift.org/origin/4.3:snapshot-provisioner quay.io/openshift/
 registry.svc.ci.openshift.org/origin/4.3:sriov-cni quay.io/openshift/origin-sriov-cni:4.3 quay.io/openshift/origin-sriov-cni:4.3.0 quay.io/openshift/origin-sriov-cni:latest
 registry.svc.ci.openshift.org/origin/4.3:sriov-dp-admission-controller quay.io/openshift/origin-sriov-dp-admission-controller:4.3 quay.io/openshift/origin-sriov-dp-admission-controller:4.3.0 quay.io/openshift/origin-sriov-dp-admission-controller:latest
 registry.svc.ci.openshift.org/origin/4.3:sriov-network-config-daemon quay.io/openshift/origin-sriov-network-config-daemon:4.3 quay.io/openshift/origin-sriov-network-config-daemon:4.3.0 quay.io/openshift/origin-sriov-network-config-daemon:latest
+registry.svc.ci.openshift.org/origin/4.3:sriov-network-webhook quay.io/openshift/origin-sriov-network-webhook:4.3 quay.io/openshift/origin-sriov-network-webhook:4.3.0 quay.io/openshift/origin-sriov-network-webhook:latest
```

Comment:
This is not in the correct location, the file is sorted alphabetically and would fail generation.

Reply:
fixed